nix/wasm-pack_0_9/default.nix

```diff
@@ -0,0 +1,48 @@
+# This is copied verbatim from nixpkgs; it is needed because the current
+# nixpkgs only supports wasm_pack 10.
+{ lib, stdenv
+, fetchFromGitHub
+, rustPlatform
+, pkg-config
+, libressl
+, curl
+, Security
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "wasm-pack";
+  version = "0.9.1";
```

Comment:
I believe we can use the latest wasm-pack 0.10.2 https://github.com/enso-org/enso/blob/a74e075adca3518cf57d0d78380b083e93f4dec7/app/gui/docs/CONTRIBUTING.md#development-environment

Reply:
Yeah, we actually must use the latest one to avoid issues on M1 Macs